packages/app/src/routes/+layout.svelte

```diff
@@ -140,6 +142,23 @@ onMount(async () => {
   await operationalStateStore.refresh();
   await packagesStore.loadInstalledPackages();
 
+  // Load Ethereum network label once for display in sidebar
+  try {
+    const cfg = await packageConfigStore.getConfig("ethereum");
+    const network = cfg.values.network;
+    if (network) {
+      if (network === "hoodi") ethereumNetworkLabel = "Hoodi";
+      else if (network === "mainnet") ethereumNetworkLabel = "Mainnet";
+      else if (network === "sepolia") ethereumNetworkLabel = "Sepolia";
+      else if (network === "ephemery") ethereumNetworkLabel = "Ephemery";
```

Comment:
I'd avoid having to maintain all the network names, especially in different places. Maybe just uppercase the first letter here?